zmon-controller-app/src/main/java/org/zalando/zmon/controller/MultiKairosDBController.java

```diff
@@ -115,6 +185,7 @@ else if (q.has("start_relative")) {
                 httpEntity, JsonNode.class);
         lf.addCallback(new StopTimerCallback(timer));
 
+        scope.close();
```

Comment:
I'm not sure this is really what we want to do. At the top, I believe we tried to obtain the scope with `Scope scope = tracer.scopeManager().active();` which suggests someone else in the call stack defined the active scope and should, therefore, be responsible to close it. I believe we should only close the scope if it was created/activated in this function.